packages/react/src/components/editor/CodeEditor.tsx

```diff
@@ -17,18 +17,48 @@ const ReactCodeMirror = loadable(() => import('./CodeMirror'), {
 });
 
 export interface ICodeEditorProps {
+    /**
+     * The unique identifier that will be used to retrieve the value from the PlasmaState
+     */
     id?: string;
+    /**
+     * The initial value
+     */
     value: string;
+    /**
+     * The mode to use as syntax highlighting in the form of a MIME type. You can use values from the CodeMirrorModes constant to help you out.
+     */
     mode: any;
+    /**
+     * When true, the content of the editor will not be editable
+     */
     readOnly?: boolean;
+    /**
+     * A callback function executed when the content of the editor changes
+     *
+     * @param code The content of the editor after the change
+     */
     onChange?: (code: string) => void;
+    /**
+     * A callback function executed when the editor is mounted to the DOM
+     */
     onMount?: (codemirror: Controlled) => void;
-    errorMessage?: string;

```

Comment:
Is it expected? If so, I think this property is still used in the component (unless `ICodeEditorProps` extends another interface that exposed this property :thinking: )

Reply:
Where?

Reply:
[packages/react/src/components/editor/JSONEditor.tsx](https://github.com/coveo/plasma/pull/2566/files#diff-1a09f9ff213de68235d8430d5dd170642ec971e38576b6bdba785268d94b988e) line 69 I believe

Reply:
Is it in the `CodeEditor` component? 😜

Reply:
oh whoops! All rights, "je retourne dans mon trou".